.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.1.4
+    working_directory: ~/haul
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          - v1-dependencies
+      - run:
+          name: update-yarn
+          command: sudo npm i -g yarn
```

Comment:
You can also use https://github.com/callstack-io/new-website/blob/master/.circleci/config.yml#L5 since that Docker build already has it (taken from react-navigation and new-website repositories)